nonlinear_elasticity/nonlinear_elasticity.cc

```diff
@@ -570,8 +570,7 @@ namespace Nonlinear_Elasticity
     dof_handler_ref.distribute_dofs(fe);
     DoFRenumbering::Cuthill_McKee(dof_handler_ref);
     DoFRenumbering::component_wise(dof_handler_ref, block_component);
-    dofs_per_block =
-      DoFTools::count_dofs_per_fe_block(dof_handler_ref, block_component);
+    DoFTools::count_dofs_per_block(dof_handler_ref, dofs_per_block, block_component);
```

Comment:
@DavidSCN I am not completely sure about this change. In deal.II 9.1, `count_dofs_per_fe_block` did not exist, but in deal.II 9.2 it is supposed to be different than `count_dofs_per_block`. I simply followed the [original reference](https://www.dealii.org/9.0.0/doxygen/deal.II/code_gallery_Quasi_static_Finite_strain_Compressible_Elasticity.html) and the code seems to run and give qualitatively same results.

Reply:
That's fine, the previous version of this function was only deprecated, since the output is handled differently in the recent version.